src/js/projects.js

```diff
@@ -706,6 +732,13 @@ function ProjectsManager(query, configurator) {
                 finished_chunks: finishedFrames
             };
 
+            var license = "License\n\nThis work is made available under a Creative Commons Attribution-ShareAlike 4.0 International License (http://creativecommons.org/licenses/by-sa/4.0/).";
```

Comment:
@cdwhitfield73  Can we place this license in a file rather than placing it directly in the code? perhaps under an `assets` directory.